Core/Core_c.php

```diff
@@ -660,3 +660,90 @@ public static function create(object $referent): WeakReference {}
      */
     public function get(): ?object {}
 }
+
+/**
+ * Stringable interface marks classes as available for serialization
+ * in a string.
+ *
+ * @since 8.0.0
+ */
+interface Stringable {
+    /**
+     * Magic method {@see https://www.php.net/manual/ru/language.oop5.magic.php}
+     * called during serialization to string.
+     *
+     * @return string
+     */
+    public function __toString(): string;
+}
+
+/**
+ * @since 8.0.0
+ */
+// TODO Uncomment after PHP 8.0 release:
+// @@Attribute(Attribute::TARGET_CLASS)
+final class Attribute {
+    /**
+     * Marks that attribute declaration is allowed only in classes.
+     *
+     * @var int
```

Comment:
Seems declaring type here doesn't make any sense as type can be easily inferred. `@var` tag here is just a noisy info. Could you please remove redundant `@var` tags in constants?